BombsHandler.py

```diff
@@ -0,0 +1,52 @@
+from Bomb import Bomb
+import pygame
+from constants import *
+
+class BombsHandler:
+
+    def __init__(self):
+        self.everyPlayersBombs = []
+        self.bombPlantedThisRound = None
+        self.isBombPlantedThisRound = False
+        self.myBombsTimers = []
+        self.maxBombsPlanted = 3
+        self.myBombsPlanted = 0
+        self.bombPower = 3
+
+    def printBombs(self, screen):
+        for bomb in self.everyPlayersBombs:
```

Comment:
maybe im just being picky, but `drawBombs` would be better in terms of our naming convention